lib/coverband/collectors/view_tracker_service.rb

```diff
@@ -8,16 +8,16 @@ module Collectors
     class ViewTrackerService < ViewTracker
       def report_views_tracked
         reported_time = Time.now.to_i
-        if views_to_record.any?
-          relative_views = views_to_record.map! do |view|
+        if @views_to_record.any?
+          relative_views = @views_to_record.map! do |view|
```

Comment:
I don't like using instance variables everywhere, but I felt like I had to do this to keep the attribute readers returning an array.

If it's ok to change their type in the exposed API, then I think that it would be better to undo the use of instance variables directly (as long as the whole code can still work with sets).

Reply:
yeah, gotcha... I guess since other interfaces call and use it, it is the safest change to do it this way avoiding breaking any external users... I am good with it.